cypress/integration/admin_institution.js

```diff
@@ -42,7 +42,8 @@ describe('User administers institutions ', () => {
     cy.get('[data-testid=customer-instituiton-save-button]').click({ force: true });
   });
 
-  it('The user should be able to upload a file for an institution', () => {
+  // TODO: unskip when backend has support for uploading file
```

Comment:
filopplasting mockes, så det skal virke uavhengig av backend? har lignende test for filopplasting på files and licenses tab

Reply:
har kommentert ut logo-komponenten i `AdminCustomerInstitutionPage.tsx`